src/UI/ListView.elm

```diff
@@ -351,6 +353,25 @@ withSelectStyle style (SelectList prop opt) =
     SelectList prop { opt | selectStyle = style }
 
 
+{-| Add id attribute to the HTML tags of the elements and the list itself.
+
+    ListView.withDomIds "wineBrands"
+        someListView
+
+Generates:
+
+    < ... id="wineBrands">
+        <... id="wineBrands-key1"></...>
+        <... id="wineBrands-key2"></...>
+        <... id="wineBrands-key3"></...>
+    </...>
+
+-}
+withDomIds : String -> ListView object msg -> ListView object msg
+withDomIds containerId (SelectList prop opt) =
```

Comment:
was I mentioned on slack, I would prefer to have `withDomId: String -> ListView -> ListView` and this only changes the parent id, I would use the `toKey` to set the children ids( in fact this maybe should be called toId :D ).

Reply:
Adding that we could `withDomId` as a standard for other components as well